web/packages/teleterm/package.json

```diff
@@ -45,9 +45,9 @@
     "@types/whatwg-url": "^13.0.0",
     "@xterm/addon-fit": "^0.10.0",
     "@xterm/xterm": "^5.5.0",
-    "electron": "34.3.0",
-    "electron-builder": "^25.1.8",
-    "electron-vite": "^3.0.0",
+    "electron": "35.1.4",
+    "electron-builder": "^26.0.12",
```

Comment:
It looks like our win target is misconfigured after the update of electron-builder.

https://github.com/gravitational/teleport.e/actions/runs/14269972249/job/40000784322#step:15:300

Reply:
I moved it to `signtoolOptions`:
>chore: remove deprecated fields from winOptions and macOptions. (For winOptions signing configuration, it has been moved to win.signtoolOptions in order to support azureOptions as a separate field and avoid bloating win configuration object. For macOptions, notarize options has been deprecated in favor of env vars for quite some time. Env vars are much more secure)